src/ui/index.tsx

```diff
@@ -2,6 +2,8 @@ import listen from "./utils/listen";
 import auth from "./utils/AuthService";
 import {CONSTANTS} from "../constants/constants";
 
+window.localStorage.clear();
```

Comment:
I think the usage of localStorage in this PR isn't justified by the desire to introduce Redux down the line. localStorage gives you a good `get/set` interface but opens the app to injection attacks and same-key collisions problems. I think one middle ground for this PR (and future Redux work) is to define a utility/singleton type class to abstract the components from the storage logic (you keep the get/set semantics of course). That utility class then can be implemented as a dictionary and if proved to be safe using a "real" storage.

Reply:
Another bit of using storage apis. SessionStorage will be guaranteed to clear after a session ends. But a dictionary buried within classes have that already.

Reply:
Yeah. Not married to Redux, by any means, but I think there's no way for us to satisfy our usability requirements (or maybe even our functional requirements) without a global store. `window.localStorage` was my solution for that problem, but I'll start exploring what a `SessionStorage` class might look like for us.

Reply:
How do you balance the need to insert the dependency of a storage api with how it was working before using a guarded dictionary that is transient. For example if you have something like: `VerifierSessionStorage.get('myKey')` you will have the same semantics and be able to power the same use cases. Using an= storage api here requires more defensive programming than the previous method. That isn't to say that the get/set really benefit the programming experience of each step.